src/NuGet/BuildNuGets.csx

```diff
@@ -280,17 +280,21 @@ int PackFiles(string[] nuspecFiles, string licenseUrl)
             p.StartInfo.Arguments = $@"{Path.Combine(ToolsetPath, "NuGet.CommandLine.XPlat.dll")} pack {file} {commonArgs}";
         }
 
+        p.StartInfo.CreateNoWindow = true;
         p.StartInfo.UseShellExecute = false;
         p.StartInfo.RedirectStandardError = true;
+        p.StartInfo.RedirectStandardOutput = true;
 
-        Console.WriteLine($"{Environment.NewLine}Running: nuget pack {file} {commonArgs}");
+        Console.WriteLine($"Packing {file}");
 
         p.Start();
         p.WaitForExit();
 
         var currentExit = p.ExitCode;
         if (currentExit != 0)
         {
+            Console.WriteLine($"nuget pack {p.StartInfo.Arguments}");
+            Console.WriteLine(p.StandardOutput.ReadToEnd());
```

Comment:
Standard workaround is you have to do the .ReadToEnd in a separate thread. Often easiest to just launch Task.Run() and do that. That ensures you have a thread reading the stream so it doesn't fill up. Then .Result the task once the .WaitForExit() returns.